.github/workflows/ci.yaml

```diff
@@ -334,7 +335,8 @@ jobs:
         run: >-
           echo "::set-output name=key::venv-${{ env.CACHE_VERSION }}-${{
             hashFiles('setup.cfg', 'requirements_test_min.txt',
-          'requirements_test_brain.txt', 'requirements_test_pre_commit.txt') }}"
+           'requirements_test_brain.txt')
+          }}"
```

Comment:
Previously, we didn't run the extra tests for `pypy` which is quite reasonable I believe.

Reply:
Also removed